pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -160,6 +160,8 @@ type TensorflowSpec struct {
 	RuntimeVersion string `json:"runtimeVersion,omitempty"`
 	// Defaults to requests and limits of 1CPU, 2Gb MEM.
 	Resources v1.ResourceRequirements `json:"resources,omitempty"`
+	// Allow user to change between REST or gRPC protocol.
```

Comment:
we will also need this for onnx, trtis model server.

Reply:
I want to use this PR to iterate and finalize our spec for the grpc protocol. For other frameworks, we can add it in a following PR.